src/CypherNet.Core/GraphStore.cs

```diff
@@ -27,7 +27,7 @@ public class GraphStore
         #region Constructors and Destructors
 
         /// <summary>
-        /// Initialises a new instance of the <see cref="GraphStore"/> class.
+        /// Initialises a new instance of the <see cref="GraphStore"/> class, usable before Neo4J version 2.2.0.
```

Comment:
I agree, the comment for the constructor is incorrect.

I'm creating an application with CypherNet.Core in an STA thread. When I get that decently working, I will add unit tests and push the changed comment. For the moment, the application hangs when executing GraphStore.Initialize.